Lotto/Lotto/StringCalculator/StringConverter.swift

```diff
@@ -47,3 +32,21 @@ struct StringConverter {
     }
     
 }
+
+extension StringConverter {
+    func emptyValueChangeZero(inputArray: [String]) -> [String] {
+        var changeArray: [String] = []
```

Comment:
```suggestion
        var changedArray: [String] = []
        //또는
        var convertedStrings: [String] = [] 
```
은 어떨까요..?! 프로퍼티 명이 동사형으로 시작해서 어색한것 같습니다 😊

[Swift API Guideline](https://www.swift.org/documentation/api-design-guidelines/#naming) 문서를 한번 읽어보셔도 좋을것 같습니다.👍🏻

Reply:
아 그렇네요! ㅎㅎ
[30b8444](https://github.com/next-step/swift-lotto/pull/30/commits/30b8444e79f72e13ffbd41f88fdf9cb1fbe8b69e)